### DISCUSSION TITLE
Add field type "Color" to builder's elements list

### DISCUSSION THREAD

**Discussion by dreiekk** [vueform/builder#59] [Ideas]
It would be great to have field type `color` to select colors with a color picker like HTML5 `<input type="color">`

**Comment by dreiekk** [comment]
I just realised, you can add a generic "Text input" field and specify a different input type in it's properties.
Maybe it's still a consideration to add a "Color" field in the Vueform Builder's elements list for easy access.

![grafik](https://github.com/vueform/vueform/assets/25348281/fb831dc1-162f-4421-990e-5e9ffe8d75d3)

**Comment by adamberecz** [comment]
Thanks for the suggestion, I'm moving this to builder Ideas.
